libhdf5/hdf5create.c

```diff
@@ -50,6 +50,10 @@ nc4_create_file(const char *path, int cmode, size_t initialsz,
     NC_HDF5_FILE_INFO_T *hdf5_info;
     NC_HDF5_GRP_INFO_T *hdf5_grp;
 
+#ifdef HAVE_H5PSET_LIBVER_BOUNDS
```

Comment:
Do we ever not have H5PSET_LIBVER_BOUNDS?

Reply:
It was introduced in HDF5-1.8.0